freedomofpress/securedrop
# [design refresh, rev2] Redesign footer, index page and language selector

## Description

Update footer and index page to the second revision as specified for the SecureDrop 1.0.0 design refresh.

Resolves #4662
Resolves #4663 

## Status

Ready for review

## Test plan

- Verify that the main page design works well in desktop and mobile view of Tor Browser
- Verify that the footer as specified appears on each page of the Source Interface
- Verify that there are no unintended side effects on any page of the Source Interface or the Journalist Interface

## Why is the logo in the footer a bit different from the main logo?

We decided to use a monochromatic version of the logo in the footer to avoid too much repetition of the exact same image (favicon, potentially stock logo if not replaced, and footer). This monochromatic variant will be one of the approved variants post-logo refresh.

## Screenshots

### Desktop view

![Screen Shot 2019-08-22 at 14 30 40-fullpage](https://user-images.githubusercontent.com/213636/63551265-7cb26180-c4e9-11e9-8868-cd96d2eb87a2.png)

### Mobile view

![Screen Shot 2019-08-22 at 14 30 53-fullpage](https://user-images.githubusercontent.com/213636/63551264-7cb26180-c4e9-11e9-8732-ed46cfbac0a0.png)
